HIP/hip-540.md

```diff
@@ -94,21 +94,42 @@ First, let's address important language to set a clear distinction between "remo
 ### Other Considerations
 
 1. We should standardize the use of all-zeros key as an invalid key across all keys in Hedera.
-2. We will have to add a boolean flag in the update transaction about whether the system should check the validity of the updated key. Currently, we test the validity for all updates. With this boolean flag, the user will be able to tell the system to avoid checking of the validity of the updated key. The default value of the flag will maintain the current behavior of checking the validity.
+2. We will have to add an enum field in the update transaction about whether the system should check the validity of the updated key. Currently, we test the validity for all updates. With this enum field, the user will be able to tell the system to avoid checking of the validity of the updated key. The default value of the field will maintain the current behavior of checking the validity.
```

Comment:
```suggestion
2. We will have to add an enum field in the update transaction about whether the system should check the validity of the updated keys. Currently, we test the validity for all updates. With this enum field, the user will be able to tell the system to avoid checking of the validity of the updated keys. The default value of the field will maintain the current behavior of checking the validity.
```